routes/cube/index.js

```diff
@@ -171,6 +174,29 @@ router.get('/view/:id', (req, res) => {
   return res.redirect(`/cube/overview/${req.params.id}`);
 });
 
+router.post('/collaborator/:id', ensureAuth, async (req, res) => {
```

Comment:
Maybe this route should be plural - `/collaborators/` - to reflect the change in functionality? That's just a nitpick though.